Rakefile

```diff
@@ -1,5 +1,7 @@
 
-#!/usr/bin/env rake
+# !/usr/bin/env rake
+# frozen_string_literal: true
```

Comment:
I see this littered through our code, but not really seen it before. Do we not need it anymore?

Reply:
Do you mean the Rakefile or this change? The Rakefile is still included in newer rails project so I guess it's still needed.

Reply:
This change. But nevermind I thought you were removing it, not adding it ...